crosscompile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cargo install cargo-hack
+
+# --feature-powerset instead of --each-feature would be possible, but would be very long
+# (by now, 128 jobs (2^7) when excluding all-authenticators and all-providers)
+cargo hack --each-feature --exclude-no-default-features build
```

Comment:
Could it be possible to add those two lines directly on the `docker run` line? Like:
```diff
-docker run -v $(pwd):/tmp/parsec -w /tmp/parsec all-providers /tmp/parsec/crosscompile.sh
+docker run -v $(pwd):/tmp/parsec -w /tmp/parsec all-providers cargo install cargo-hack && cargo hack --each-feature --exclude-no-default-features build
```
or add them as a new command of the `ci.sh` script?

I am just a bit cautious about having too many testing scripts at the root, for clarity. But maybe it's fine!